lib/models/network/abstract_classes/abstract_account_model.dart

```diff
@@ -1,24 +1,49 @@
 import 'package:defi_wallet/models/address_book_model.dart';
+import 'package:defi_wallet/models/balance/balance_model.dart';
 import 'package:defi_wallet/models/history_model.dart';
+import 'package:defi_wallet/models/network/abstract_classes/abstract_network_model.dart';
 import 'package:defi_wallet/models/network/network_name.dart';
 import 'package:defi_wallet/models/token_model.dart';
 
 abstract class AbstractAccountModel {
   final String publicKey;
   final Map<NetworkName, String> addresses;
+  final Map<NetworkName, List<BalanceModel>> pinnedBalances;
   final int accountIndex;
 
-  AbstractAccountModel(this.publicKey, this.addresses, this.accountIndex);
+  AbstractAccountModel(
+      this.publicKey, this.addresses, this.accountIndex, this.pinnedBalances);
```

Comment:
need to format for long strings for every params in single line. I will use * for the same issue